documentation/developers/rfid/generic_nfcpy.md

```diff
@@ -0,0 +1,24 @@
+# Generic NFCPy Reader
+
+This module is based on the user space NFC reader library [nfcpy](https://nfcpy.readthedocs.io/en/latest/overview.html) ([on github](https://github.com/nfcpy/nfcpy)).
+The link above also contains a list of supported devices.
```

Comment:
If there’s a direct link to the supported devices available, this could be helpful.

Reply:
I could link to the anchor as well, (like https://nfcpy.readthedocs.io/en/latest/overview.html#supported-devices)